challenge3/webapp/msaito/db/seeds.rb

```diff
@@ -0,0 +1,29 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
+#   Character.create(name: 'Luke', movie: movies.first)
+
+require "csv"
+
+CSV.foreach('db/house_data.csv', headers: true) do |row|
+  House.create!(:id => row[0],
```

Comment:
```suggestion
  House.create!(id: row[0],
```
と Ruby 1.9以降の記法を使う方がいい。